app/utils/notification/index.ts

```diff
@@ -90,12 +90,22 @@ export const emitNotificationError = (type: 'Team' | 'Channel' | 'Post' | 'Conne
 
 export const scheduleExpiredNotification = (serverUrl: string, session: Session, serverName: string, locale = DEFAULT_LOCALE) => {
     const expiresAt = session?.expires_at || 0;
-    const expiresInDays = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asDays()));
+    const expiresInHours = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asHours()));
```

Comment:
Out of the scope for this PR but @enahum are we taking the expire information from the right place?

I have the feeling this is calculating the time between now and the expire at of the session, not the session length configured in the server.

Am I missing something?

Reply:
So we are using the session expiredAt property that is set on the server, when you first login it will be basicslly `now + session lenght` and the next time ut csn be different if extended sessions is enabled on the server, that is why we cancel the scheduled notification, get the session info from the server and schedule another one each time the app is dent to the background if I'm not mistaken. This value is basically a timestamp that is then use for the scheduled local notification.

Reply:
Oh! I see now. Thanks!